apps/schema/src/components/ColumnEditModal.vue

```diff
@@ -114,11 +114,18 @@
             </div>
             <div class="row">
               <div class="col-4" v-if="isEditable(column)">
-                <InputBoolean
-                  id="column_required"
-                  v-model="column.required"
+                <InputRadio
+                  id="column_required_radio"
                   label="required"
+                  :options="[true, false, 'condition']"
+                  v-model="requiredSelect"
```

Comment:
ik kijk vast niet goed, maar hoe komt als 'true' or 'false' de waarde in column.required terecht?

Reply:
@update:modelValue="setRequired" dus in deze functie wordt ie gezet